docs/developer-setup/automated-releases.mdx

```diff
@@ -1,22 +1,37 @@
 # Releases
 
+The OpenSRP FHIR Core releases occur at most once every 2 weeks, i.e. at the conclusion of a sprint. The release process follows the [gitlab flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html#release-branches-with-gitlab-flow) style described in the following diagram:
+
+![gitlab flow diagram](../assets/gitlabflow.png)
+
+This allows changes to occur on the release branch while unrelated code continues being merged into main. Specifically, release follow the following steps:
+1. Code freeze: the creation of a branch for the release version as well as the first release candidate (labeled RC1).
+    * For example, for release version v0.2.0 a branch opensrp-0.2.0 is create and a pre-release opensrp-0.2.0-rc1
+1. Candidate progression: This is followed by 1-2 weeks of QA and error fixing
+    * Whenever we find an error in the candidate, a pull request with a fix for the error is opened against the release branch (e.g. v0.2.0) and against the main branch.
+    * Once the error is fixed and merged into the release branch (multiple errors can be batched) a new pre-release is created, e.g. opensrp-0.2.0-rc2
+1. Final release: when the release meets requirements, a final release, opensrp-0.2.0 is created, with a tag v0.2.0.
```

Comment:
We can update the doc to distinguish between **Github tagged releases** and actual **APK releases** for disambiguation

Reply:
k update to split into 2 sections